[FOCS] Convert species from txt to py

Cjkjvfnby opened the pull request:
- COnvert tech that have fill support in Game

o01eg reviewed (changes requested):
Tests get failed with: 
```
  2: [2023-07-06 23:21:36.018503] [0x000003f4] [info]    Couldn't find file for module spec focs
  2: [2023-07-06 23:21:36.018503] [0x000003f4] [info]    Traceback (most recent call last):
  2: [2023-07-06 23:21:36.018503] [0x000003f4] [info]      File "<string>", line 1, in <module>
  2: [2023-07-06 23:21:36.018503] [0x000003f4] [info]    ModuleNotFoundError: No module named 'focs'
  2: [2023-07-06 23:21:36.018503] [0x000003f4] [info]    Unable to parse data file D:\a\freeorion\freeorion\build\Release\test-scripting\species\SP_ABADDONI.focs.py
```

You should copy `focs` to `test-scripting` because some tests use it as root for its python modules.

o01eg reviewed (changes requested):
There is difference in dump: [1.txt](https://github.com/freeorion/freeorion/files/11988720/1.txt)

I'll fix `SetCapacity` in #4605  but it different in `NO_OFFENSE_TROOPS_DESC` effect

```diff
                 Source
             activation =
-                And [
-                    Ship
-                    Or [
+                Or [
+                    And [
+                        Ship
                         DesignHasPart name = "GT_TROOP_POD"
-                        DesignHasPart name = "GT_TROOP_POD_2"
-
                     ]
+                    DesignHasPart name = "GT_TROOP_POD_2"
+
                 ]
             accountinglabel = "NO_OFFENSIVE_TROOPS_LABEL"
             stackinggroup = "NO_OFFENSIVE_TROOPS_STACK"
```

Also it would be fine to squash changes.

o01eg reviewed (changes requested):
`SP_CELESTEPHYTE`: Had `SetCapacity partname = "GT_TROOP_POD" value = 0.000000` became `SetCapacity partname = "GT_TROOP_POD" value = Value`
`SP_CRAY`: Bad industry still misses: 
```
                    Planet
                    TargetIndustry low = 0.000000
                    Happiness low = 0.000000
```
`SP_CYNOS`: Good.
`SP_DERTHREAN`: Had `SetCapacity partname = "GT_TROOP_POD" value = 0.000000`, became `SetCapacity partname = "GT_TROOP_POD" value = Value`
`SP_EGASSEM`: Good.

Cjkjvfnby committed:
Convert species

Cjkjvfnby committed:
Fix the typo in the human file

o01eg reviewed (changes requested):
There were changes in Influence and Research focus about Exobots

Cjkjvfnby committed:
Port changes in macroses

Cjkjvfnby committed:
Convert fulver

Cjkjvfnby committed:
Add empty activation to match macros code

Cjkjvfnby committed:
Replace formulas with result.

This make some noise in diff, Python code evaluates this expression on early stage and they are not preserved in parsing.
